python_modules/libraries/dagster-airflow/dagster_airflow/cli.py

```diff
@@ -105,7 +104,7 @@ def main():
 @click.option(
     "--module-name", "-m", type=click.STRING, help="The name of the source module", required=True
 )
-@click.option("--pipeline-name", type=click.STRING, help="The name of the pipeline", required=True)
+@click.option("--pipeline-name", type=click.STRING, help="The name of the name")
```

Comment:
This should be the "The name of the pipeline", not "The name of the name", right?

Reply:
err thanks for catching!